### PR TITLE
EE-1128: add chain_get_era_info RPC endpoint

### PR DESCRIPTION
**Pull request by henrytill**
Ref: https://casperlabs.atlassian.net/browse/EE-1128

(needed for client integration)

### CODE REVIEW

**Review by EdHastingsCasperLabs — APPROVED** [review]

File: client/lib/lib.rs [hunk 1]
```diff
@@ -341,6 +341,25 @@ pub fn get_balance(
...
+/// Retrieves era information from the network.
+///
+/// * `maybe_rpc_id` is the JSON-RPC identifier, applied to the request and returned in the
+///   response. If it can be parsed as an `i64` it will be used as a JSON integer. If empty, a
+///   random `i64` will be assigned. Otherwise the provided string will be used verbatim.
+/// * `node_address` is the hostname or IP and port of the node on which the HTTP service is
+///   running, e.g. `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the JSON-RPC request will be printed to `stdout`.
+/// * `maybe_block_id` must be a hex-encoded, 32-byte hash digest or a `u64` representing the
+///   `Block` height or empty. If empty, the latest `Block` transfers will be retrieved.
```
Comment (Fraser999) [inline client/lib/lib.rs:353]: This comment should speak of era info rather than transfers.

**Review by Fraser999 — APPROVED** [review]
One tiny nitpick, but otherwise LGTM.

File: client/lib/lib.rs (hunk 1 above)
Comment (henrytill) [reply client/lib/lib.rs:353]: Whoops, some bad copy-pasta there, good catch :+1:

**Pull request merged**